pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java

```diff
@@ -79,6 +79,28 @@ public Response registerFunction(final @PathParam("tenant") String tenant,
         return functions.registerFunction(
             tenant, namespace, functionName, uploadedInputStream, fileDetail, functionDetailsJson);
     }
+    
+    
+    @POST
+    @ApiOperation(value = "Creates a new Pulsar Function in cluster mode")
+    @ApiResponses(value = {
+            @ApiResponse(code = 403, message = "The requester doesn't have admin permissions"),
+            @ApiResponse(code = 400, message = "Invalid request (function already exists, etc.)"),
+            @ApiResponse(code = 408, message = "Request timeout"),
+            @ApiResponse(code = 200, message = "Pulsar Function successfully created")
+    })
+    @Path("/{tenant}/{namespace}/{functionName}/url")
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    public Response registerFunction(final @PathParam("tenant") String tenant,
```

Comment:
this change seems to be based on other PRs?

Reply:
yes, let me rebase this PR once the #1903 is merged.